app.py

```diff
@@ -3,27 +3,9 @@
 import pandas as pd
 from datetime import datetime, date
 
-# Front matter
-st.set_page_config(
-    page_title="dbt Model Finder",
-    page_icon=":shark:",
-    layout="centered",
-    initial_sidebar_state="auto",
-    menu_items=None,
-)
-st.title("dbt Model Finder :shark:")
-st.subheader("(aka: find the model that's still running)")
-st.write(
-    "I love [dbt](https://www.getdbt.com)! But often when a dbt run hangs, I'm searching through the output line-by-line trying to figure out which models are _still_ running. 😵"
-)
-st.write(
-    "Well, never again. 🎉 Now there's an app for that! Paste your in-progress dbt output in the textbox below. The text can be copied from your command line interface or from the dbt Cloud UI. Log files _downloaded_ from dbt Cloud are not supported yet."
-)
-
-with st.expander("dbt output example"):
-    st.code(
-        """
-    10:03:09  1 of 10 START sql table model hyrule.source_quests  [RUN]
+
+REGEX = r"\s*(?:\[0m)?(\d{2}:\d{2}:\d{2})\s+(\d+)\s+of\s+\d+\s+.*?(?:model\s|of\s)\b(\w+\.\w+)\b"
```

Comment:
General comment about the regex approach taken here. [In the first release of this script](https://github.com/foundinblank/dbt-model-finder/blob/11e07522b95e2013d8fcf8df1f330d5f1b1b0065/app.py) I used a regex approach too! Then I noticed that the output was very systematic and that by dropping certain words (`extraneous_words`), I could parse based on space separators and output into a data frame and do the rest there. 

Your PR suggests returning to the regex approach and I'm not sure I want to. Want to walk me through your reasoning some more?